content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md

```diff
@@ -0,0 +1,158 @@
+### Introduction
+Databases hold a lot of critical and highly sensitive data for various organizations. For this reason, it is crucial to monitor them and hence make sure they are securely accessed. The article will aim at enlightening the reader on how to monitor a Microsoft SQL Server database using Solar Winds Database Performance Analyzer (Self-Managed). This is a software offered by Solar Winds and is very useful in attaining the security of a database.
+
+### Table of Contents
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Installing the DPA on windows](#installing-the-dpa-on-windows)
+- [How to register MySQL Server for Monitoring](#how-to-register-mysql-server-for-monitoring)
+- [How to register the database instances for monitoring](#how-to-register-the-database-instances-for-monitoring)
+- [Best practices](#best-practices)
+- [Conclusion](#conclusion)
+### Prerequisites
+1.	Correct and needed credentials
+2.	Correct system requirements
+3.	A powerful server where the DPA (Database performance analyzer) will be installed.
+4.	A repository database
+5.	Customer portal account to help access licenses and downloads. The customer portal can be accessed using this [link](https://customerportal.solarwinds.com/).
+6.	A local administrator account to be used by the server hosting DPA.
+7.	Updated operating system.
+8.	Open the following ports on the server and firewall: 8124, 8123, 8127, 443, 80, and 587.
+9.	Exclude the DPA directory from antivirus scanning to ensure smooth installation: C:\Program Files\SolarWinds\DPA
+
+Note: DPA is short for Database Performance Analyzer.
+
+### Phase 1
+
+### Installing the DPA on windows
+1.	If you're installing a trial version, [click here](https://www.solarwinds.com/database-performance-analyzer) or the SolarWinds Customer Portal to get the installation file.
+2.	Log in to the Windows server using the administrator account.
+3.	Extract the ZIP file you downloaded into a traceable folder.
+
+![File Extraction](/engineering-education/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/File Extraction.png)
```

Comment:
I would suggest using image names separated by `-`, and all the letters in small case.

Reply:
Could you do this change for all the images?